package.js

```diff
@@ -1,22 +1,26 @@
 Package.describe({
   summary: 'Style with attitude. Sass and SCSS support for Meteor.js.',
-  version: '4.15.1',
+  version: '4.16.0',
   git: 'https://github.com/Meteor-Community-Packages/meteor-scss.git',
   name: 'fourseven:scss',
 });
 
 Package.registerBuildPlugin({
   name: 'compileScssBatch',
-  use: ['caching-compiler@1.2.2 || 2.0.0', 'ecmascript@0.15.1'],
+  use: [
+    'caching-compiler',
+    'ecmascript'
+  ],
   sources: ['plugin/compile-scss.js'],
   npmDependencies: {
-    'node-sass': '4.14.1',
-  },
-});
+    "node-sass": '9.0.0',
+    "@babel/runtime": "7.24.5"
+  }
+})
 
 Package.onUse(function(api) {
-  api.versionsFrom('2.3');
-  api.use('isobuild:compiler-plugin@1.0.0');
```

Comment:
Removing the version constraint here works locally, but prevents publication. The question is which version should we use for Meteor 3. Has it been updated or is it still 1.0.0?

Reply:
nice catch, I shall check in my application and update this PR.

Reply:
I left this one alone and it worked fine for `leonardoventurini:scss` in Meteor 3, I think I searched for an example in the Meteor repo.